robots.json

```diff
@@ -524,13 +524,27 @@
         "operator": "[Sidetrade](https://www.sidetrade.com)",
         "respect": "Unclear at this time."
     },
+    "SummalyBot": {
+        "description": "AI input summarization.",
+        "frequency": "No information.",
+        "function": "Extracts data to provide summaries for user; additional usages unclear.",
+        "operator": "[Grammarly](https://www.grammarly.com/ai/ai-writing-tools/summarizing-tool) (suspected not definitively known at this time)",
```

Comment:
What evidence is there to suspect Grammarly? Unless there is something tangible, I'd prefer to play it safe and say "Unknown.".